getting-started/quickstart-guides/angular.md

```diff
@@ -0,0 +1,834 @@
+    }];
+    lqip = { 
+        active: true, quality: 20
+    };
+    ...
+}
+```
+{% endcode %}
+
+{% code title="src/app/app.component.html" %}
+```js
+<ik-image
+    path={{path}}
+    urlEndpoint="https://ik.imagekit.io/demo/"
+    [transformation]="transformation"
+    [lqip]="lqip"
+    >
+  </ik-image>
+```
+{% endcode %}
+
+### **Combining lazy loading with low-quality placeholders**
+
+You have the option to lazy-load the original image only when the user scrolls near them. Until then, only a low-quality placeholder is loaded. This saves a lot of network bandwidth if the user never scrolls further down.
+
+{% code title="src/app/app.component.ts" %}
+```jsx
+    ...
+    transformation: Array<Transformation> = [{
+        height:"300",
+        width: "400",
+    }];
+    lqip = { 
+        active: true
+    };
+    ...
+}
+```
+{% endcode %}
+
+{% code title="src/app/app.component.html" %}
+```js
+// Loading a blurred low quality image placeholder 
+// and lazy-loading original when user scrolls near them
+<ik-image
+    path={{path}}
+    urlEndpoint="https://ik.imagekit.io/demo/"
+    [transformation]="transformation"
+    [lqip]="lqip"
+    loading="lazy"
+    >
+  </ik-image>
+```
+{% endcode %}
+
+## **Uploading files in Angular**
+
+Let's now learn how to upload an image to our media library.
+
+Angular SDK provides `ik-upload` component which renders an `input type="file"` tag that you can use to upload files to the [ImageKit media library](../../media-library/overview/) directly from the client-side.
+
+To implement this functionality, a backend server is needed to authenticate the request using your [API private key](../../api-reference/api-introduction/api-keys.md#private-key).
+
+### **Setup the backend app**
+
+For this quickstart guide, we will create a sample Node.js server which will provide an authentication endpoint at `http://localhost:3000/auth`.&#x20;
+
+Let's a new folder `server` and create files `app.js` and `package.json` inside the new folder in the project root.
+
+```bash
+mkdir server
+touch server/app.js
+```
+
+Package.json should look like:
+```js
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Sample server for file upload using Imagekit SDK",
+  "main": "app.js",
+  "scripts": {
+    "server": "nodemon app"
+  },
+  "devDependencies": {
+    "cors": "^2.8.5",
+    "crypto": "^1.0.1",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.2",
+    "router": "^1.3.3",
+    "uuid": "^3.3.3"
+  }
+}
+```
+
+Now do `npm install`.
+
+Next we setup the content for `app.js`.
+
+{% tabs %}
+{% tab title="app.js" %}
+{% code title="server/app.js" %}
+```javascript
+const dotenv = require('dotenv');
+const express = require('express');
+const router = express.Router();
+var cors = require('cors');
+const app = express();
+app.use(cors());
+
+dotenv.config();
+
+const uuid = require('uuid');
+const crypto = require("crypto");
+
+const privateKey = process.env.PRIVATE_KEY;
```

Comment:
We have not mentioned anywhere in this guide that this value needs to be replaced. This code won't work as it is and needs explanation in this guide.

Reply:
Adding the need to include .env file with PRIVATE_KEY

Reply:
Are we reading from `.env` in the the server app? If not why not simply do like React sample server.